app/ui/lib/Modal.tsx

```diff
@@ -56,13 +57,9 @@ export function Modal({ children, onDismiss, title, isOpen }: ModalProps) {
               modal={false}
             >
               <Dialog.Portal>
-                <div
-                  className="DialogOverlay pointer-events-auto"
-                  onClick={onDismiss}
-                  aria-hidden
-                />
+                <DialogOverlay />
```

Comment:
I can't find any problem with removing this `onClick` and the `pointer-events-auto`. Everything works – clicking out to close the modal, scrolling popovers when they're inside the modal, etc.